### ISSUE TITLE
Check python version for RPI aarch64 

### ISSUE THREAD

**Issue by pergolafabio** [pergolafabio/Hikvision-Addons#48]
Make sure the python version is not updated on openeuler,  the SDK doesnt work on all versions, on amd64 its now sticky to 3.10.8

@mion00 do you have an RPI?

**Comment by mion00** [comment]
Yep I have a Raspberry PI with 64bit OS

**Comment by pergolafabio** [comment]
Ah, have you tested the openeuler? Does the SDK still work ?  Cause the python recently updated to 3.10.9 on the Python-slim container, that's why I choose 3.10.8-slil image... Maybe is better to choose a specific tag for openeuler too?

**Comment by mion00** [comment]
No for now I have only tested on my local dev machine (x64 laptop).
Yes I saw that you were using the latest tag. Maybe it is better to use one of the LTS versions of openeuler?

**Comment by pergolafabio** [comment]
Indeed, but don't have the hardware to test

**Comment by mion00** [comment]
Ah ok so you only have x86?

**Comment by pergolafabio** [comment]
Indeed, I created that addon based on speculation :-)

**Comment by mion00** [comment]
I cannot run it as an addon since I don't have the HA supervisor, but only as a docker container. Ok I'll try to find a way to run on my RPi anyway.

**Comment by pergolafabio** [comment]
Yeah, I used the original script in beginning, it was a clean   small script with IP/user all hardcoded in hik.py
